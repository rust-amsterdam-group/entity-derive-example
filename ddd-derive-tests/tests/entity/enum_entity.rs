use ddd_derive::Entity;

#[derive(Entity)]
enum Status {
    OPEN,
    CLOSED,
    PENDING,
}

fn main() {}