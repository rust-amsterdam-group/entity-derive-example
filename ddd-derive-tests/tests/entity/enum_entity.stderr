error: Entity cannot be derived from Enum

  = note: Entity may only be derived from a Struct with named fields
  = help: Try changing the item to a Struct with named fields

 --> $DIR/enum_entity.rs:4:1
  |
4 | / enum Status {
5 | |     OPEN,
6 | |     CLOSED,
7 | |     PENDING,
8 | | }
  | |_^
